### PR TITLE
feat(packages/kaizenplatform) Added integration 

### LINKED ISSUE #438: kaizenplatfowm integration
# TL;DR

before refs: http://docs.userdive.com/ja/web/devguide/integration/kaizenplatform/

- [ ] Fixed namespace
- [ ] Fixed validation
- [ ] Documents

## Backward compatibility

nothing

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Closes #438

### CODE REVIEW

File: packages/kaizenplatform/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@userdive/kaizenplatform-plugin",
+  "description": "webpage tracking agent linker plugin",
```
Comment (9renpoto) [inline packages/kaizenplatform/package.json]: need spec

File: packages/kaizenplatform/src/plugin.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  }
+  getVariation () {
+    (q('kzs', window) as any)(
+      'getVariation',
+      ({ variationId }: { variationId: string }, state: string) => {
+        try {
+          if (state === 'decided') {
+            this.tracker.send('event', {
+              eventCategory: 'kaizenplatform',
+              eventLabel: `?_kzs_var_id=${variationId}`
```
Comment (Surgo) [inline packages/kaizenplatform/src/plugin.ts]: Send only `variationId` and specify that type using `eventAction`

File: packages/kaizenplatform/src/plugin.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import Agent from '@userdive/agent'
+import { q } from 'userdive'
+
+export default class Plugin {
+  tracker: Agent
+  constructor (tracker: Agent) {
+    this.tracker = tracker
+  }
+  getVariation () {
+    (q('kzs', window) as any)(
```
Comment (shoota) [inline packages/kaizenplatform/src/plugin.ts]: kaizenplatform code will inject `kzs` under window.
https://support.kaizenplatform.net/hc/en-us/articles/206075262-Collecting-Variation-data

File: package.json [hunk 4]
```diff
@@ -118,6 +118,10 @@
     {
       "path": "cdn/linker.js",
       "limit": "1.2 KB"
+    },
+    {
+      "path": "cdn/kaizenplatform-plugin.js",
+      "limit": " 700 B"
```
Comment (shoota) [inline package.json]: plugin size is 653 Byte now

File: packages/kaizenplatform/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@userdive/kaizenplatform-plugin",
+  "description": "a plugin for webpage tracking with kaizenplatform",
+  "version": "0.0.1",
+  "author": "UNCOVER TRUTH Inc.",
+  "bugs": {
+    "url": "https://github.com/userdive/agent.js/issues"
+  },
+  "dependencies": {
+    "@userdive/agent": "^1.0.0-beta5.3",
```
Comment (shoota) [inline packages/kaizenplatform/package.json]: old

File: packages/kaizenplatform/src/plugin.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import Agent from '@userdive/agent'
+import { q } from 'userdive'
+
+export default class Plugin {
+  private tracker: Agent
+  constructor (tracker: Agent) {
+    this.tracker = tracker
+  }
+  public getVariation () {
+    (q('kzs', window) as any)(
```
Comment (shoota) [inline packages/kaizenplatform/src/plugin.ts]: kaizenplatform code will inject `kzs` under window.
https://support.kaizenplatform.net/hc/en-us/articles/206075262-Collecting-Variation-data

**Pull request merged**